torch/_functorch/partitioners.py

```diff
@@ -2141,3 +2277,95 @@ def draw_graph(
         write_method(fname)
     else:
         write_method(fname, prog=prog)
+
+
+def should_quantize(node: torch.fx.Node) -> bool:
+    allowed_dtypes = [
```

Comment:
could also do: `dtype.is_floating_point and dtype.itemsize > torch.float8_e5m2.itemsize`